config/data/treatment-inertia/clean_medicine_to_dosage.csv

```diff
@@ -0,0 +1,50 @@
+rxcui,medicine,dosage

```

Comment:
Could consider sourcing these medication names and rxnorm codes from `medications.yml`. It'll reduce duplication.

Reply:
There's [a story](https://app.shortcut.com/simpledotorg/story/2886/integrate-etl-pipeline-with-existing-drug-name-dosage-cleanup-work) for this, and I'll look into it. Not all the drugs in `medications.yml` have dosages, and they aren't floats. Needs some additional thought and work. We might want to convert `medications.yml` into a table and use that for both purposes, perhaps? Not sure.